packages/eslint-config-terra/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "eslint-config-terra",
+  "version": "3.10.0",
+  "description": "Terra Eslint Config",
+  "main": "eslint.config.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/eslint-config-terra"
+  },
+  "keywords": [
+    "eslint",
+    "eslint-config",
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/eslint-config-terra/issues"
+  },
+  "engines": {
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
+  },
+  "homepage": "https://github.com/cerner/eslint-config-terra",
+  "scripts": {
+    "clean": "rm -rf node_modules",
+    "clean:install": "npm run clean && npm install",
```

Comment:
we might not need these guys when comparing to terra-core: https://github.com/cerner/terra-core/blob/master/packages/terra-button/package.json#L24

Reply:
https://github.com/cerner/terra-toolkit/pull/402/commits/81534d2ce1c15eac348e43573e0b42148d28f719